cms/urls.py

```diff
@@ -41,6 +41,7 @@
 
     url(r'^not_found$', 'contentstore.views.not_found', name='not_found'),
     url(r'^server_error$', 'contentstore.views.server_error', name='server_error'),
+    url(r'^org_names_list$', OrganizationListView.as_view(), name='org_names_list'),
```

Comment:
As with my comment above, it would be ideal to model this as "/organizations", if possible